### PR TITLE
Feature transaction spending

### PR DESCRIPTION
**Pull request by MelvinDunn**

### CODE REVIEW

File: usaspending_api/search/v2/elasticsearch_helper.py [hunk 1]
```diff
@@ -224,3 +224,38 @@ def get_transaction_ids_agg(keyword, size=10000):
         for result in response['aggregations']['results']['buckets']:
             results.append(result['key'])
         yield results
+
+
+def get_sum_and_count_aggregation_results(keyword):
+    index_name = '{}-'.format(TRANSACTIONS_INDEX_ROOT.replace('_', ''))+'*'
```
Comment (tony-sappe) [inline usaspending_api/search/v2/elasticsearch_helper.py]: Why is the string replace needed? The index root is the prefix for all transaction indices

Comment (MelvinDunn) [reply usaspending_api/search/v2/elasticsearch_helper.py]: it's not

File: usaspending_api/search/v2/elasticsearch_helper.py [hunk 2]
```diff
@@ -224,3 +224,38 @@ def get_transaction_ids_agg(keyword, size=10000):
...
+                 }
+              },
+              "prime_awards_count": {
+                 "value_count": {
+                    "field": "transaction_id"
+                 }
+              }
+              }, "size": 0}
+    found_result = False
+    while not found_result:
```
Comment (tony-sappe) [inline usaspending_api/search/v2/elasticsearch_helper.py]: Why is there a while loop?

File: usaspending_api/search/v2/elasticsearch_helper.py [hunk 3]
```diff
@@ -241,11 +241,11 @@ def get_sum_and_count_aggregation_results(keyword):
                  }
               }
               }, "size": 0}
-    found_result = False
-    while not found_result:
+    found_result = 0
+    while not found_result == 10:
```
Comment (tony-sappe) [inline usaspending_api/search/v2/elasticsearch_helper.py]: Better to have `found_result >= 10`

File: usaspending_api/search/v2/elasticsearch_helper.py [hunk 4]
```diff
@@ -224,3 +224,38 @@ def get_transaction_ids_agg(keyword, size=10000):
...
+                 "value_count": {
+                    "field": "transaction_id"
+                 }
+              }
+              }, "size": 0}
+    found_result = 0
+    while not found_result == 10:
+        try:
+            response = CLIENT.search(index=index_name, body=query)
+            found_result += 1
```
Comment (tony-sappe) [inline usaspending_api/search/v2/elasticsearch_helper.py]: Need to move this outside of the try:except block. it is possible the first statement errors and found_result never gets incremented

**Pull request merged**